### DISCUSSION TITLE
Degenerate configuration

### DISCUSSION THREAD

**Discussion by joshuajaistein** [neperfepx/neper#437] [Q&A] [answered]
Hello sir , I am trying to input seeds manually using morphooptini command. But even on randomising I am getting the Degenerate configuration error. 

How to solve this ?

neper -T -n 40 -morphooptiini "coo:file(Carves.txt2)" -domain "cube(5,5,1)" -periodicity all -format tesr,vtk -tesrsize 50:50:10 -o jjgeometr
[Carves.txt](https://github.com/rquey/neper/files/8466099/Carves.txt)
y

**Comment by rquey** [comment, marked as answer]
@joshuajaistein, please do not open several discussions for the same problem.

In your file, several seeds have the same x, y or z coordinate. They are therefore not randomized...

Duplicates #427 and #435.
